web/src/components/SaveMessage.test.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { shallow } from "enzyme";
+import moment from "moment";
+import SaveMessage from "./SaveMessage";
+
+describe("<SaveMessage />", () => {
+  let lastSaved;
+  let subject;
```

Comment:
```suggestion
```

I'd move these closer to the point-of-use, at which point they can be `const`s. For me, at least, it's clearer that the value is new and I'm not worrying about overwriting some previously existing variable.